guides/common/modules/proc_enabling-client-connections-to-satellite.adoc

```diff
@@ -1,16 +1,14 @@
-[id="enabling-client-connections_{context}"]
-[id="enabling-client-connections-to-satellite_{context}"]
+[id="Enabling_Connections_from_a_Client_to_Server_{context}"]
 = Enabling Connections from a Client to {ProjectServer}
 
 {SmartProxies} and Content Hosts that are clients of a {ProjectServer}'s internal {SmartProxy} require access through {Project}'s host-based firewall and any network-based firewalls.
 
-Use this procedure to configure the host-based firewall on the Red Hat Enterprise Linux 7 system that {Project} is installed on, to enable incoming connections from Clients, and to make the configuration persistent across system reboots.
+Use this procedure to configure the host-based firewall on your {ProjectServer} to enable incoming connections from Clients and to make the configuration persistent across system reboots.
```

Comment:
I remember something about how we really wanted to emphasise how this action needed to happen on the base OS on which you were installing Foreman/Satellite. I think you are obfuscating the point with your change.

Reply:
OK, IMHO the `ProjectServer` attribute implies that (in contrast to `Project`). But I've mostly reverted my change. Now, all it removes is `Red Hat Enterprise Linux 7` because Foreman and orcharhino also runs on EL7 clones.

Reply:
Is this ok for you @melcorr ?